Makefile

```diff
@@ -155,13 +155,10 @@ souffle:
 	    "Fedora Linux") \
 	      sudo dnf -y install https://github.com/souffle-lang/souffle/releases/download/2.5/x86_64-fedora-41-souffle-2.5-Linux.rpm;; \
 	    "Ubuntu") \
-	      sudo wget https://souffle-lang.github.io/ppa/souffle-key.public -O /usr/share/keyrings/souffle-archive-keyring.gpg; \
-	      echo "deb [signed-by=/usr/share/keyrings/souffle-archive-keyring.gpg] https://souffle-lang.github.io/ppa/ubuntu/ stable main" | sudo tee /etc/apt/sources.list.d/souffle.list; \
 	      sudo apt update; \
-          sudo wget https://github.com/souffle-lang/souffle/releases/download/2.5/x86_64-ubuntu-2404-souffle-2.5-Linux.deb; \
-		  sudo apt install ./x86_64-ubuntu-2404-souffle-2.5-Linux.deb; \
-		  rm x86_64-ubuntu-2404-souffle-2.5-Linux.deb; \
-	      sudo apt install souffle;; \
+	      sudo wget https://github.com/souffle-lang/souffle/releases/download/2.5/x86_64-ubuntu-2404-souffle-2.5-Linux.deb; \
+	      sudo apt install ./x86_64-ubuntu-2404-souffle-2.5-Linux.deb; \
```

Comment:
This `deb` file is intended for Ubuntu 22.04. When I install it in Ubuntu 22.04 I got this issue
```
The following packages have unmet dependencies:
 souffle : Depends: libc6 (>= 2.38) but 2.35-0ubuntu3.9 is to be installed
           Depends: libstdc++6 (>= 13.1) but 12.3.0-1ubuntu1~22.04 is to be installed
```
But using this file https://github.com/souffle-lang/souffle/releases/download/2.5/x86_64-ubuntu-2204-souffle-2.5-Linux.deb works. 

I'm okay with fixing this issue in a different PR if we are intended to support multiple Ubuntu versions.

Reply:
We should still support Ubuntu 22.04 LTS as devs might need more time to upgrade to 24.04 LTS. @benmss Can you please address this issue in this PR?